### ISSUE TITLE
Manual Install clarification

### ISSUE THREAD

**Issue by storepeinture** [EaDesgin/magento2-pdf-generator2#10]
Hello EADesign-Team,

when trying to install with composer, I'm running into the following "fatal error":
`Fatal error: Out of memory (allocated 814743552) (tried to allocate 67108864 bytes) in /composer.phar/src/Composer/DependencyResolver/Solver.php on line 220
`

Now I have decided to install manually and downloaded the package from your website.

Your manual installation instruction is very brief:
**You can install the module [...] manually by adding it to the app/code directory.**
But when I look at the package folder structure, it looks more like it should be placed into the **vendor-folder**.

I'm confused as to what exactly shall be placed into the app/code directory.
What would be the proper SSH-commands to execute afterwards? (run `upgrade` and then `deploy`??)

In other words, can you be a bit more precise on the manual installation instructions.
Thank you for clarifying!

Best,
Michael

**Comment by eadesignro** [comment]
Hi @storepeinture ,

The issues you reported is a server configuration issue not a extension issue. You need to tweak your server settings. Usually the PHP memory.

**Comment by storepeinture** [comment]
Ok thank you for the info! Unfortunately I cannot change it because my php memory is limited. I cannot increase php memory more.

That's why I'm asking for info on manual installation.

Can you give some clarification on how to do manual installation.

**Comment by storepeinture** [comment]
I don't understand why you closed my issue after 20 minutes, without even giving a full response to my question. And now ... no answer anymore ...
